.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 ################
 
 # Git Hub integration and bot rules
-/.github/                                       @AlexGhiondea @jsquire
+/.github/                                           @AlexGhiondea @jsquire
```

Comment:
@AlexGhiondea, @jsquire, do you want to scope this to the `fabricbot.json` file so you aren't pinged for review when CODEOWNERs change?

Reply:
@alzimmermsft:  Please do not change this.   We need visibility on CODEOWNERS changes in order to keep the bot rules up to date with "Service Attention" contacts or when paths for PR labels change.

Reply:
Got it, didn't realize it was tied closely to CODEOWNERS

Reply:
Unfortunately, it is - and its currently a manual sync due to bot limitations.